wsd/COOLWSD.cpp

```diff
@@ -1551,6 +1555,33 @@ class RemoteConfigPoll : public RemoteJSONPoll
         }
     }
 
+    void fetchMonitors(std::map<std::string, std::string>& newAppConfig,
+                       Poco::JSON::Object::Ptr remoteJson)
+    {
+        Poco::JSON::Array::Ptr monitors;
+        try
+        {
+            monitors = remoteJson->getArray("monitors");
+        }
+        catch (const Poco::NullPointerException&)
+        {
+            LOG_INF("Overriding monitor failed because array "
+                    "does not exist");
+            return;
+        }
+
+        if (monitors.isNull() || monitors->size() == 0)
+        {
+            LOG_INF("Overriding monitors failed because array is empty or "
+                    "null");
+            return;
+        }
+
+        for (size_t i = 0; i < monitors->size(); i++)
```

Comment:
Does this suffer from an ever growing number of duplicate monitors ? if the list shrinks does this work nicely or leave older things away ? (or perhaps newAppConfig is just the new data we're over-writing with?)